### DISCUSSION TITLE
Complex number support

### DISCUSSION THREAD

**Discussion by chrisrichardson** [wjakob/nanobind#234] [Q&A] [answered]
I would like to use complex numbers with `ndarray` which seems to be supported by DLpack, and was something that could be done with `pybind11`. Is there any plan to enable it? Are there any likely problems?

**Comment by wjakob** [comment, marked as answer]
I don't plan to do it myself, but you are welcome to look into it and propose a PR. I think it might involve quite a bit of work to get it to correctly interact with the various tensor frameworks.
